start-distributed

```diff
@@ -3,7 +3,7 @@
 DYNO_CONNECT_DISTRIBUTED_PROPERTIES_FILE="dyno-connect-distributed.properties"
 DEBEZIUM_CONNECTOR_URL="https://repo1.maven.org/maven2/io/debezium/debezium-connector-postgres/0.7.2/debezium-connector-postgres-0.7.2-plugin.tar.gz"
 
-./setup_certs HEROKU_KAFKA_JCHAO
+./setup_certs KAFKA
```

Comment:
This is a good change. Thanks for this. Previously, I had my environment set to this key when I worked at Heroku.